cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -193,7 +193,7 @@ def add_flow_cell_data_to_housekeeper(
 
     def add_sample_fastq_files(self, flow_cell_directory: Path, flow_cell_name: str) -> None:
         """Add sample fastq files from flow cell to Housekeeper."""
-        fastq_file_paths: List[Path] = self.get_sample_fastq_file_paths(
+        fastq_file_paths: List[Path] = self.get_flowcell_sample_fastq_file_paths(
```

Comment:
Maybe get fastq_file_paths_for_flow_cell_and_sample?

Reply:
Nice catch, the naming can be improved! It retrieves the fastq files for all samples on the flow cell, so maybe get_all_flow_cell_sample_fastq_files is more descriptive

Reply:
Decided on `get_all_sample_fastq_file_paths_for_flow_cell`, let me know if you think it can be made even more clear